src/NuGet.Clients/NuGet.CommandLine/Commands/SignCommand.cs

```diff
@@ -189,13 +189,12 @@ private void ValidateCertificateInputs(ILogger logger)
             }
             else if (CertificateFingerprint != null)
             {
-                if (!CertificateUtility.TryDeduceHashAlgorithm(CertificateFingerprint, out HashAlgorithmName hashAlgorithmName))
+                if (!CertificateUtility.TryDeduceHashAlgorithm(CertificateFingerprint, out HashAlgorithmName hashAlgorithmName) ||
```

Comment:
If we merge this now, it'll be 6.13 that starts failing. Is that the intent?

Reply:
That is correct, Nikolche.